### DISCUSSION TITLE
Google Analytics Connector

### DISCUSSION THREAD

**Discussion by joshgk00** [vanjarosoftware/Vanjaro.Platform#2148] [Q&A] [answered]
I see there's a Google Analytics connector in Vanjaro. The field is labeled Measurement Id. Is that the same as the GA account Id that's usually in the similar form of UA-xxxxxxx-x ? 

Is there place where I can see the script where that value will be used?

**Comment by meetmandeep** [comment, marked as answer]
Measurement ID is used with GA 4 (newer version of analytics). Next update of Vanjaro re-introduces support for older version of Google Analytics as well (UA-xxxxxxx)

**Comment by joshgk00** [comment]
Thanks for the clarification.
